### PR TITLE
UE-2075: Free space for Jenkins/Report to add new sequences

### PR DESCRIPTION
**Pull request by quangnt**
@thuhuyenit @PhuongHNguyen @kieudiem1989 @zelonght Can you have a look on this?

### CODE REVIEW

**Comment by zelonght** [comment]
@quangnt Commit text not follow convention

**Comment by zelonght** [comment]
@quangnt Next time The PR should clearly tell what is if for ([like this](https://github.com/PersonifyInc/StagePresence/pull/1468#issue-170744475)), because it helps other (not in the same page) to review it easier.

File: git-fat [hunk 1]
```diff
@@ -273,7 +273,12 @@ class GitFat(object):
         self.verbose = verbose_stderr if os.environ.get('GIT_FAT_VERBOSE') else verbose_ignore
         self.gitroot = subprocess.check_output('git rev-parse --show-toplevel'.split()).strip()
         self.gitdir = subprocess.check_output('git rev-parse --git-dir'.split()).strip()
-        self.objdir = os.path.join(self.gitdir, b'fat', b'objects')
+        cfgpath = os.path.join(self.gitroot,b'.gitfat')
+        objdir = gitconfig_get('s3.objdir', file=cfgpath)
```
Comment (zelonght) [inline git-fat]: I think you should add instruction to here https://github.com/PersonifyInc/git-fat#s3-remote and FAT repo later too. 

@quangnt We need to always stake care the documentation to help other to use the new changes

**Comment by quangnt** [comment]
@zelonght Already addressed comments. Currently add note in Readme.md(Section "S3 remote"). Will update Readme.md on Fat later

File: README.md [hunk 2]
```diff
@@ -58,6 +58,7 @@ Edit your .gitfat file and add the following:
     bucket={bucketname}
     key={access_key_id}
     secret={secret_access_key}
+	objdir={stored_git_fat_object_folder}
```
Comment (thuhuyenit) [inline README.md]: Spacing issue. @quangnt Next time: you should self review to avoid that.

**Comment by thuhuyenit** [comment]
@quangnt Had a draft commit https://github.com/PersonifyInc/git-fat/pull/11/commits/fa381b60c269c381c2f0554cb7f60156ef9ed21a. I think you should squash 3 commit to 1 and commit should be UE-2075: Fee space for Jenkins/Report to add new sequences. It help github link to Jira card

**Comment by quangnt** [comment]
@thuhuyenit Already address your comment. @zelonght Will also create commit in FAT to update Readme.md

**Comment by zelonght** [comment]
re: "@thuhuyenit Already address your comment. @zelonght Will also create commit in FAT to update Readme.md"  - follow convention, any comment address must have a reply. Please check again.

File: git-fat (hunk 1 above)
Comment (quangnt) [inline git-fat]: @zelonght Already fixed this comment. Will address in FAT later

File: README.md [hunk 3]
```diff
@@ -58,6 +58,7 @@ Edit your .gitfat file and add the following:
     bucket={bucketname}
     key={access_key_id}
     secret={secret_access_key}
+    objdir={stored_git_fat_object_folder}
```
Comment (zelonght) [inline README.md]: `stored_git_fat_object_folder`  --> `location_to_store_git_fat_object`

Add a commit for this.

**Comment by zelonght** [comment]
:+1:

**Comment by thuhuyenit** [comment]
:+1: For the commit of LongTruong https://github.com/PersonifyInc/git-fat/pull/11/commits/719b0f0eb26c72d2010c4244ab17ae846063048f
@quangnt Please squash on the commit. Thanks!

**Comment by quangnt** [comment]
@zelonght @thuhuyenit Done(Squash into 1 commit). Can you review?

**Comment by zelonght** [comment]
Merging...

**Pull request merged**